### ISSUE TITLE
readinessProbe & livenessProbe?

### ISSUE THREAD

**Issue by billimek** [munnerz/kube-plex#39]
There is currently no readiness or liveness probes.  Can we consider adding something like this?

```
        readinessProbe:
          tcpSocket:
            port: pms
          initialDelaySeconds: 5
          periodSeconds: 10
        livenessProbe:
          tcpSocket:
            port: pms
          initialDelaySeconds: 15
          periodSeconds: 20
```

**Comment by billimek** [comment]
Fixed via #48

**Comment by scruplelesswizard** [comment]
@billimek can you close?
